operations/exports.md

```diff
@@ -0,0 +1,158 @@
+# Exports
+
+> ### Restricted!
+> These operations are currently in beta-test and as such are subject to change.
+
+## Get all exports
+
+Get exports by provided `ExportIds`. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/exports/getAll`
+
+```json
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ExportIds": [
+        "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+        "09708665-0e31-4b23-b337-b0a000be0df0",
+        "706dc6d5-9511-4751-825e-538ce99da2ce",
+        "f776f20a-6f1a-4ddf-93f4-9dae95261415"
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ExportIds` | array of string | optional, max 1000 items | Unique identifiers of the exports. If not specified, the operation returns exports with Status of `Pending`, `Processing`, and `Success` for the enterprise associated with the Access Token. |
+| `EntityType` | string [Entity type](#entity-type) | optional | Return only exports with the provided [Entity type](#entity-type). |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of message data returned (using cursor pagination). |
+
+### Response
+
+```json
+{
+    "Exports": [
+        {
+            "Id": "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+            "Status": "Success",
+            "EntityType": "OrderItem",
+            "ExpiresUtc": "2023-10-26T11:42:28Z",
+            "Files": [
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-1.jsonl?example=signature",
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                },
+            ]
+        },
+        {
+            "Id": "09708665-0e31-4b23-b337-b0a000be0df0",
+            "Status": "Pending",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
+            "Status": "Processing",
+            "EntityType": "OrderItem",
+            "Files": [],
+            "ExpiresUtc": null
+        },
+        {
+            "Id": "706dc6d5-9511-4751-825e-538ce99da2ce",
```

Comment:
```suggestion
            "Id": "f776f20a-6f1a-4ddf-93f4-9dae95261415",
```
Wrong Id I guess? It's duplicated with the one above.

Reply:
Right, pasted the wrong one, good catch!